packages/sui-studio/bin/sui-studio-generate.js

```diff
@@ -67,79 +67,90 @@ Promise.all([
   writeFile(
   COMPONENT_PACKAGE_GITIGNORE_FILE,
   `lib
-  node_modules
-  `
-  ),
+node_modules`),
 
   writeFile(
   COMPONENT_PACKAGE_NPMIGNORE_FILE,
-  `src
-  `
-  ),
+  `src`),
```

Comment:
maybe you could add the assets folder

Reply:
which is the assets folder ?!

Reply:
we are adding previews (screenshots) for the readme in the assets folder

Reply:
It's just a comment (optional)